package.json

```diff
@@ -17,6 +17,7 @@
     "heroku-postbuild": "npm run build",
     "ledger-server": "NODE_PATH=dist node --optimize_for_size --max_old_space_size=460 --gc_interval=100 dist/ledger/server.js",
     "ledger-worker": "NODE_PATH=dist node --optimize_for_size --max_old_space_size=460 --gc_interval=100 dist/ledger/worker.js",
+    "lint": "standard --fix",
```

Comment:
in every other brave repo, `lint` is mapped to `standard`. i recommend calling this `lint-fix` or something